app/views/components/tag.test.js

```diff
@@ -23,46 +23,51 @@ const createDummyApp = (context) => {
 };
 
 describe('foundation-tag', () => {
-  it('should render the foundation tag when isFoundation is true', (done) => {
+  it('should render the tag with the correct data-test-id', (done) => {
     const context = {
-      isFoundation: true,
+      qaIdentifier: 'qa-identifier',
     };
     const dummyApp = createDummyApp(context);
     request(dummyApp)
       .get('/')
       .then((res) => {
         const $ = cheerio.load(res.text);
-        const foundationTag = $('[data-test-id="qaidentifier-foundation-tag"]');
-        expect(foundationTag.length).toEqual(1);
-        expect(foundationTag.hasClass('nhsuk-u-margin-bottom-6')).toEqual(true);
-        expect(foundationTag.hasClass('bc-c-tag-foundation')).toEqual(true);
-        expect(foundationTag.text().trim()).toEqual('Foundation Solution');
+        expect($('div[data-test-id="qa-identifier-tag"]').length).toEqual(1);
+        // expect(tag.hasClass('bc-c-tag')).toEqual(true);
+
+        // const tag = $('.bc-c-tag');
```

Comment:
remove the comments if they are no longer required

Reply:
yeahhhh I think we should add this to the linter - I will take a look